dev/VSIX/ProjectTemplates/Desktop/CSharp/UnitTestApp/UnitTests.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.AppContainer;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace $safeprojectname$
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
+        public void TestMethod1()
+        {
+            Assert.AreEqual(0, 0);
+        }
+
+        // Use the UITestMethod attribute for tests that need to run on the UI thread.
+        [UITestMethod]
+        public void TestMethod2()
+        {
+            var grid = new Grid();
```

Comment:
not the Grid in markup?

Reply:
I just wanted to demonstrate the simplest unit test that needs to run on the UI Thread as a way to demo `        [UITestMethod]` attribute.

Reply:
does it make sense to x:Name the grid in markup and use it instead? otherwise, the markup looks like cruft, and we don't have as realistic/complete a scenario

Reply:
nope, leave it - agree with you